server/db_setup.sql

```diff
@@ -9,7 +9,7 @@ create table laptop_request (
     lastname varchar(100),
     email varchar(100),
     phonenumber varchar(100),
-    laptop_request_status laptop_request_status DEFAUlT 'ACTIVE'
+    laptop_request_status  DEFAUlT 'ACTIVE'
```

Comment:
I think this is wrong?

Reply:
fixed

Reply:
I think it's still wrong. As best I can tell, `laptop_request_status laptop_request_status DEFAUlT 'ACTIVE'` is correct for that line

Reply:
I fixed back